docs/jwst/wavecorr/description.rst

```diff
@@ -9,6 +9,11 @@ The wavelength correction (``wavecorr``) step in the
 assignments for NIRSpec fixed-slit (FS) and MOS point sources that are
 known to be off center (in the dispersion direction) in their slit.
 
+Upon successful completion of the step, the status keyword "S_WAVCOR"
+in the primary header is set to "COMPLETE".  For each SCI extension, the "WAVECOR"
+keyword is set to True if the slit was wavelength corrected (it is a point
+source) or False if it was not corrected (it is a point source).
```

Comment:
both of these say "(it is a point source)" I think there's supposed to be a NOT in one?